data/discovery/field/citizens-advice-bureau.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: citizens-advice-bureau
+phase: discovery
+register: citizens-advice-bureau
+text: A Citizens Advice Bureau
```

Comment:
`A Citizens Advice Bureau in England or Wales`.  Perhaps the membership number should be used for this field, in which case the `citizens-advice-bureau-membership-number` field could be removed.